CSS/home style/home_grid.css

```diff
@@ -1,30 +1,45 @@
 .grid {
     display: grid;
-    grid-template-columns: 1fr 1fr 1fr;
-    row-gap: 3rem;
-    column-gap: 3rem;
+    justify-content: center;
+    grid-template-columns: 25rem 25rem 25rem;
+    grid-gap: 3rem;
+    margin-bottom: 2rem;
 }
 
 .grid__box {
-    display: block;
+    display: flex;
+    align-items: center;
+    flex: 1;
+    flex-direction: column;
+    text-align: center;
     font-size: 1.8rem;
 }
 
 .grid__imgContainer {
-    display: flex;
-    align-items: center;
-    width: 8rem;
-    height: 8rem;
-    padding-left: 6rem;
+    max-width: 8rem;
 }  
 
 .grid__imgContainer img {
     width: 100%;
 }
 
 .grid__boxTitle {
-    display: block;
     font-size: 2.2rem;
     padding-top: 1.5rem;
     padding-bottom: 2rem;
-}
\ No newline at end of file
+}
+
+/* table size */
+@media screen and ( max-width: 1000px) {

```

Comment:
We will automate this soon, but until then make sure your spacing between is consistent

Reply:
sure